README.md

```diff
@@ -0,0 +1,22 @@
+# Merl-an
+
+> Tool for [merlin](https://github.com/ocaml/merlin) performance analysis
+
+Merl-an generates queries to test [merlin](https://github.com/ocaml/merlin)
+performance, executes them and gathers the results.
+
+## Local development
+
+Since **merl-an** streamlines Merlin's main trunk, we recommend building a local
+switch and manually adding the dependency to Merlin:
+
+```shell
+# opam update
+# opam switch create . ocaml-base-compiler.4.14.1 --deps-only -y
```

Comment:
Which compiler version is needed will depend on the state of merlin's main branch. So probably the best is to create an empty switch and have the compiler version be solved by opam.

Reply:
Since adding merlin `pinned` happens after Switch initialization, I'm not sure if it works.

Reply:
I think it should work. You can create an empty switch via
```
opam switch create --empty <switch-name>
```
(or a local switch, if you prefer that)
Afterwards, you pin merlin, which will than automatically also install the correct OCaml version into the switch as it isn't installed yet:
```
opam pin https://github.com/ocaml/merlin.git
```

Let me know if I've misunderstood you!